torch/csrc/distributed/rpc/py_rref.cpp

```diff
@@ -13,6 +13,17 @@ namespace rpc {
 /////////////////////  Pickle/Unpickle Helplers ////////////////////////////
 
 namespace {
+
+std::shared_ptr<FutureIValue> toFutureIValue(
+    std::shared_ptr<FutureMessage> fm) {
+  auto fv = std::make_shared<FutureIValue>();
+
+  fm->addCallback(
+      [fv](const FutureMessage& fm) { fv->markCompleted(IValue()); });
```

Comment:
Nit: we also don't propagate the error here, do we need to do that?

Reply:
It doesn't seem we need that for the current usage, as it seems that the returrned `FutureIValue` is only used by the profiler, which only uses the callback feature on the Future. But let me pass the error to the new Future as we are going to move this to a utility function.